Build failure with time >= 1.11

```
src/Data/Store/Internal.hs:800:30: error:
    • Not in scope: type constructor or class ‘Time.FirstWeekType’
      Neither ‘Data.Time’, ‘Data.Time.Calendar.Quarter’,
              ‘Data.Time.Clock.System’,
              ‘Data.Time.Clock.TAI’ nor ‘Data.Time.Format.ISO8601’ exports ‘FirstWeekType’.
    • In the Template Haskell quotation
        [d| instance Store a => Deriving (Store (Complex a))
            instance Store a => Deriving (Store (Identity a))
            instance Deriving (Store All)
            instance Deriving (Store Any)
            instance Deriving (Store Void)
            instance Deriving (Store Bool)
            instance (Store a, Store b) => Deriving (Store (Either a b))
            instance Deriving (Store Time.AbsoluteTime)
            instance Deriving (Store Time.Day)
            instance Deriving (Store Time.LocalTime)
            instance Deriving (Store Time.TimeOfDay)
            instance Deriving (Store Time.TimeZone)
            instance Deriving (Store Time.UTCTime)
            instance Deriving (Store Time.UniversalTime)
            instance Deriving (Store Time.ZonedTime)
            instance Deriving (Store Time.TimeLocale)
            instance Deriving (Store Time.SystemTime)
            instance Deriving (Store Time.CalendarDiffDays)
            instance Deriving (Store Time.CalendarDiffTime)
            instance Deriving (Store Time.FormatExtension)
            instance Deriving (Store Time.DayOfWeek)
            instance Deriving (Store Time.FirstWeekType)
            instance Deriving (Store Time.Quarter)
            instance Deriving (Store Time.QuarterOfYear) |]
      In the untyped splice:
        $(derive
            [d| instance Store a => Deriving (Store (Complex a))
                instance Store a => Deriving (Store (Identity a))
                instance Deriving (Store All)
                instance Deriving (Store Any)
                instance Deriving (Store Void)
                instance Deriving (Store Bool)
                instance (Store a, Store b) => Deriving (Store (Either a b))
                instance Deriving (Store Time.AbsoluteTime)
                instance Deriving (Store Time.Day)
                instance Deriving (Store Time.LocalTime)
                instance Deriving (Store Time.TimeOfDay)
                instance Deriving (Store Time.TimeZone)
                instance Deriving (Store Time.UTCTime)
                instance Deriving (Store Time.UniversalTime)
                instance Deriving (Store Time.ZonedTime)
                instance Deriving (Store Time.TimeLocale)
                instance Deriving (Store Time.SystemTime)
                instance Deriving (Store Time.CalendarDiffDays)
                instance Deriving (Store Time.CalendarDiffTime)
                instance Deriving (Store Time.FormatExtension)
                instance Deriving (Store Time.DayOfWeek)
                instance Deriving (Store Time.FirstWeekType)
                instance Deriving (Store Time.Quarter)
                instance Deriving (Store Time.QuarterOfYear) |])
    |
800 |     instance Deriving (Store Time.FirstWeekType)
    |                              ^^^^^^^^^^^^^^^^^^
cabal: Failed to build store-0.7.12
```

--- a/src/Data/Store/Internal.hs
+++ b/src/Data/Store/Internal.hs
@@ -142,6 +142,7 @@ import qualified Data.Time.Format.ISO8601 as Time
 #endif
 #if MIN_VERSION_time(1,11,0)
 import qualified Data.Time.Calendar.Quarter as Time
+import qualified Data.Time.Calendar.WeekDate as Time
 #endif
 
 #ifdef INTEGER_GMP

